internal/db/migrations/2023-09-29.0-alter-receiver_wallets-add-invitation_sent_at-column.sql

```diff
@@ -0,0 +1,9 @@
+-- +migrate Up
+
+ALTER TABLE public.receiver_wallets
+    ADD COLUMN invitation_sent_at timestamp with time zone;
+
```

Comment:
🚩Your migration should also populate that column with a non null value for receiver wallets who were already invited. Otherwise, you will end up triggering new SMS for existing users who already received an invitation.

Please, keep in mind this product is already running in production and any changes can affect real users.